### DISCUSSION TITLE
Rabbitmq connections not increasing more than 129k in 3.13.3 and Erlang 26.2.5.2. #11735

### DISCUSSION THREAD

**Discussion by lakshyasharma1196** [rabbitmq/rabbitmq-server#11744] [Questions] [answered]
Hi Team,
Recently I upgrading from 3.11 to 3.13. For that load testing is performing. Rabbitmq is running in cluster node with 12 nodes.
We have noticed that our connections are not increasing more than 1,29,000 . We have increased the number of rabbitmq nodes also increased jmeter nodes. But number is not increasing for connections. is there any parameter that needed to be add or change

cat /etc/rabbitmq/rabbitmq.conf
listeners.tcp.1 = 0.0.0.0:5672
mqtt.listeners.tcp.1 = 0.0.0.0:1443
#mqtt.tcp_listen_options.backlog = 9182
mqtt.tcp_listen_options.recbuf  = 191072
mqtt.tcp_listen_options.sndbuf  = 191072
mqtt.tcp_listen_options.keepalive = true
mqtt.tcp_listen_options.nodelay   = true
mqtt.tcp_listen_options.exit_on_close = true
mqtt.tcp_listen_options.send_timeout  = 120
# anonymous connections, if allowed, will use the default
# credentials specified here
mqtt.allow_anonymous  = false
mqtt.default_user     = guest
mqtt.default_pass     = guest
mqtt.vhost            = /
mqtt.exchange         = amq.topic
# 24 hours by default
mqtt.max_session_expiry_interval_seconds = 86400
#mqtt.prefetch         = 10
management.listener.port = 15672
management.listener.ip   = 0.0.0.0

 
###### Disk & Memory Configuration ######
disk_free_limit.absolute = 50MB
vm_memory_high_watermark.relative = 0.9
######## Logging #######
log.file.level = debug
handshake_timeout = 20000
tcp_listen_options.backlog = 50000
tcp_listen_options.recbuf  = 191072
tcp_listen_options.sndbuf  = 191072
tcp_listen_options.keepalive = true
tcp_listen_options.nodelay   = true
mqtt.tcp_listen_options.backlog = 20000

**Comment by michaelklishin** [comment, marked as answer]
@lakshyasharma1196 please [read this dedicated section in the docs](https://www.rabbitmq.com/docs/networking#tuning-for-large-number-of-connections).
